### PR TITLE
Fixes Cameras by giving them all planes.

### PR DESCRIPTION
**Pull request by dwasint**
## About The Pull Request
Forgot about this with emissives before, as this was fixed downstream already.
Now all planes are added to cameras on use

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
fix:cameras display proper again
/:cl:

### CODE REVIEW

**Comment by itsmeow** [comment]
Can you check if this works on the supplypod launcher as well which also uses ByondUi?

**Comment by dwasint** [comment]
> Can you check if this works on the supplypod launcher as well which also uses ByondUi?

image

**Comment by dwasint** [comment]
image
better image they do infact work

File: code/_onclick/hud/plane_master.dm [hunk 1]
```diff
@@ -24,12 +24,6 @@
 	blend_mode = BLEND_MULTIPLY
 	alpha = 255
 
-/atom/movable/screen/plane_master/openspace/backdrop(mob/mymob)
```
Comment (PowerfulBacon) [inline code/_onclick/hud/plane_master.dm:27]: Why this change?

**Review by PowerfulBacon — DISMISSED** [review]

File: code/_onclick/hud/plane_master.dm (hunk 1 above)
Comment (dwasint) [reply code/_onclick/hud/plane_master.dm:27]: its redundant, this is already being done on init

**Comment by Fronsis** [comment]
Speedmerge when, this visual glitch pains any job that requires camera viewing 😭

**Review by itsmeow — APPROVED** [review]
